src/provider.rs

```diff
@@ -102,6 +120,13 @@ impl TryFrom<Element> for StatusLineElement {
 
     fn try_from(value: Element) -> Result<Self, Self::Error> {
         let mut transformations = vec![];
+        if value.case_from.is_some() || value.case_to.is_some() {
```

Comment:
This will be easier when changing the case-instruction to a separate struct. Something like:

```rust
if let Some(case_conversion) = value.case_conversion {
    transformations.push(StatusLineTransformation::ChangeCase(str_to_case(case_conversion.from),
        str_to_case(case_conversion.to)));
}
```
Once `str_to_case` returns a `RahmenResult`, you can handle errors with `str_to_case(...)?`.